31.12.2020 20:29:42 Checking for system tray...
31.12.2020 20:29:42 Starting pgAdmin4 server...
31.12.2020 20:29:42 Creating server object, port:53483, key:dafb4640-f6aa-4509-b107-8154bcbee195, logfile:C:/Users/bouss/.pgAdmin4.2123502176.log
31.12.2020 20:29:42 Python Path: C:/Program Files/PostgreSQL/10/pgAdmin 4/venv/Lib/site-packages;C:/Program Files/PostgreSQL/10/pgAdmin 4/venv/DLLs;C:/Program Files/PostgreSQL/10/pgAdmin 4/venv/Lib
31.12.2020 20:29:42 Python Home: C:/Program Files/PostgreSQL/10/pgAdmin 4/venv
31.12.2020 20:29:42 Initializing Python...
31.12.2020 20:29:43 Python initialized.
31.12.2020 20:29:43 Adding new additional path elements
31.12.2020 20:29:43 Redirecting stderr...
31.12.2020 20:29:43 stderr redirected successfully.
31.12.2020 20:29:43 Initializing server...
31.12.2020 20:29:43 Webapp Path: C:/Program Files/PostgreSQL/10/pgAdmin 4/web/pgAdmin4.py
31.12.2020 20:29:43 Server initialized.
31.12.2020 20:29:43 Starting Server Thread...
31.12.2020 20:29:43 Open the application code and run it.
31.12.2020 20:29:43 Set the port number, key and force SERVER_MODE off
31.12.2020 20:29:43 PyRun_SimpleFile launching application server...
31.12.2020 20:29:44 Application Server URL: http://127.0.0.1:53483/?key=dafb4640-f6aa-4509-b107-8154bcbee195
31.12.2020 20:29:44 The server should be up, we'll attempt to connect and get a response. Ping the server
31.12.2020 20:30:36 Attempt to connect one more time in case of a long network timeout while looping
31.12.2020 20:30:36 Everything works fine, successfully started pgAdmin4.
